frontend/public/features.ts

```diff
@@ -93,10 +94,18 @@ const detectOpenShift = dispatch => coFetchJSON(openshiftPath)
       : handleError(err, FLAGS.OPENSHIFT, dispatch, detectOpenShift)
   );
 
+const projectListPath = `${k8sBasePath}/apis/project.openshift.io/v1/projects`
+const detectProjectsAvailable = dispatch => coFetchJSON(projectListPath)
+  .then(
+    res => setFlag(dispatch, FLAGS.PROJECTS_AVAILABLE, _.size(res.items) > 0),
+    err => handleError(err, FLAGS.PROJECTS_AVAILABLE, dispatch, detectProjectsAvailable)
```

Comment:
Is this going to keep trying repeatedly on a non-OpenShift cluster?

Reply:
I think this only gets run once on the first page load. Although, this means your first comment about setting the flag when a user creates a project is true. We will need to reset this flag when a user creates the first project or deletes the last one.

Reply:
Ahh, I see what you mean about retrying repeatedly. It does look like that would happen. Maybe we should add a 404 check like the OpenShift flag?